lib/libsh/libsimbol/libsimbol.sh

```diff
@@ -987,6 +1023,7 @@ declare g_CACHED=${g_CACHED?}
 declare g_LDAPHOST=${g_LDAPHOST?}
 declare g_FORMAT=${g_FORMAT?}
 declare g_TLDID=${g_TLDID?}
+declare g_SSH_OPTS="${g_SSH_OPTS?}"
```

Comment:
Good catch, but yes - this is inside a block that's *generating* a shell script that's to be sourced by another piece of code which has no access to `g_SSH_OPTS`.